### PR TITLE
A bit more details for Scenarios IDs

### PR DESCRIPTION
**Pull request by rsvoboda**
A bit more details for Scenarios IDs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,4 +27,11 @@ The root cause is that the standard JDK code can't handle properly `sun.net.www.
...
-```
\ No newline at end of file
+```
+
+### Running on RHEL 7 or macOS
+On some platforms users may experience `Caused by: java.lang.IllegalStateException: Can not connect to Ryuk` errors when running modules with Testcontainers dependencies.
+
+Details can be found all over the internet, e.g. https://github.com/testcontainers/testcontainers-java/issues/572, https://www.testcontainers.org/features/configuration/#disabling-ryuk.
+
+Workaround is to use `TESTCONTAINERS_RYUK_DISABLED=true mvn test`
```
Comment (bsideup) [inline README.md:37]: This section is misleading. There are no issues with Testcontainers and Mac OS X with the latest release of Testcontainers (there was a regression caused by Docker For Mac's behaviour change)

Disabling Ryuk is a very dangerous recommendation that should not be given unless fully understood.

Issue 572 was caused by a wrong network configuration and not the library itself, so I would not refer to it.
There are currently no known issues with RHEL nor Mac OS X, as long as Docker is properly installed and configured.

Comment (rsvoboda) [reply README.md:37]: Ok, I will try https://mvnrepository.com/artifact/org.testcontainers/testcontainers/1.15.0

Comment (rsvoboda) [reply README.md:37]: Tried 1.15 on macOS with latest Docker Desktop 2.5 and it works well, thanks for pointer @bsideup.

**Comment by rsvoboda** [comment]
`Testcontainers and RHEL 7 or macOS` commit dropped

**Review by mjurc — APPROVED** [review]

**Pull request merged**